draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -1112,6 +1112,8 @@ Using the same key material in different, incompatible ways risks cross-protocol
 
 To reduce the risk of attacks if this guidance is not followed, the LabeledWindow structure defined in {{signing}} includes a label string, and the CA's `issuer_id`. Extensions of this protocol MAY be defined which reuse the keys, but any that do MUST use a different label string and analyze the security of the two uses concurrently.
 
+Furthermore, any valid signature of a CAs `public_key` that is not of the form as defined in {{signing}}, or an extension of this protocol, MUST be considered a misissuance.
```

Comment:
Extremely nitpicky nitpick: "MUST be considered misissuance" feels like kind of a weird directive. (What does it mean if I direct you to "consider" something as something else?) I think something like "CA's MUST NOT do x" and "this is considered a misissuance" reads a bit better. (Then we're just saying that this document considers such behavior a misissuance, as more a definition, than a directive.)

To that end, the previous paragraph says "To avoid this, {{parameters}} forbids the reuse of Merkle Tree CA private keys in another protocol", which I think refers to:

> To prevent cross-protocol attacks, the key used in a Merkle Tree CA MUST be unique to that Merkle Tree CA. It MUST NOT be used in another Merkle Tree CA, or for another protocol, such as X.509 certificates.

{{signing}} also says:

> If the CA's private key signs an input that can be interpreted as a LabeledWindow structure, the CA is considered to have [...]

WDYT if, instead, we appended to the first paragraph something like:

> To avoid this, {{parameters}} forbids the reuse of Merkle Tree CA private keys in another protocol. A CA MUST NOT generate signatures with its private key, except as defined in {{signing}}, or an extension of this protocol. Any valid signature of a CA's `public_key` that does not meet these requirements is considered a misissuance.

(Hmm, is it "misissuance" if my private key gets stolen by an attacker and then the attacker generates a bad certificate, or do we use a different word for that? @devonobrien )

Reply:
sgtm

Reply:
Whoops, @devonobrien and I talked about "misissuance" here and I learned that term is more nuanced than I thought. But I forgot to follow-up here with the conclusion, and now I've forgotten. Devon, do you remember / do you have thoughts on how to say this?

Reply:
(Tweaked slightly discussing with Devon.)